src/solvers/refinement/string_constraint_generator.h

```diff
@@ -365,15 +375,16 @@ class string_constraint_generatort: messaget
   string_exprt empty_string(const refined_string_typet &ref_type);
 };
 
-exprt is_digit_with_radix(const exprt &chr, const exprt &radix);
+exprt is_digit_with_radix(
+  const exprt &chr, const exprt &radix_char_type, unsigned long radix=0);
```

Comment:
I find it confusing that `radix_char_type` is not a `typet`. Maybe change it to `radix_as_char`.